src/libstd/lib.rs

```diff
@@ -241,6 +241,7 @@
 #![feature(bind_by_move_pattern_guards)]
 #![feature(box_syntax)]
 #![feature(c_variadic)]
+#![feature(cfg_if)]
```

Comment:
I think that the `pub use` from core will have to be updated below to include `cfg_if` as well